app/mixins/textable.rb

```diff
@@ -0,0 +1,21 @@
+module Textable 
```

Comment:
I am curious as to the decision to make this a module rather than a Service. I think either can work, just want to make sure you can speak to your design decision. Does pulling all this info into another class (via include) make sense over a service? I could see arguments for both sides.